Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
 github "JanGorman/Hippolyte" "0.6.0"
 github "httpswift/swifter" "1.4.6"
-github "hyperwallet/hyperwallet-ios-sdk" "1.0.0-beta01"
+github "hyperwallet/hyperwallet-ios-sdk" "22c4bc9b177eabc9277614cef95d3c2daf4ac7e1"
```

Comment:
should we change the carthage package version?

Reply:
We changed the `Cartfile` to target the `Development` branch in order to keep sync with `hyperwallet-ios-sdk`.  So the `Cartfile.resolved` file will be updated all the time.